src/tools/crossgen2/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -126,14 +128,31 @@ private static bool RegisterJITModule()
             }
         }
 
+        private IntPtr JitLibraryResolver(string libraryName, System.Reflection.Assembly assembly, DllImportSearchPath? searchPath)
+        {
+            IntPtr libHandle = IntPtr.Zero;
+            if (libraryName == JitLibrary)
+            {
+                libHandle = NativeLibrary.Load(_jitConfig.JitPath, assembly, searchPath);
```

Comment:
Before we add actual dependencies on NetCoreApp3.0 APIs: does VS debugging work for you?

I couldn't get F5 debugging to work in VS ever since @davidwrighton made crossgen2 a netcoreapp3.0 app. I just work around it locally by downgrading it back to netcoreapp2.1. I never had the time to figure out if this is my local issue or it's broken for everyone. This is going to break my hack, making this more urgent to resolve.

Reply:
No issues with debugging with VS 2019 for me. F5 works as expected.

Reply:
Cc @trylek who is hitting the same issue in VS when trying to launch crossgen2 as a netcoreapp3.0 app.